scan/spec/test_command_generator_spec.rb

```diff
@@ -521,6 +521,27 @@
                                          :test
                                        ])
         end
+
+        it "uses the correct build command with the example project when using Xcode 11 or later and a custom result bundle path", requires_xcodebuild: true do
+          allow(FastlaneCore::Helper).to receive(:xcode_version).and_return('11')
+          log_path = File.expand_path("~/Library/Logs/scan/app-app.log")
+
+          options = { project: "./scan/examples/standard/app.xcodeproj", result_bundle_path: './my_test_output/custom 2024-09-30 at 5.10.35 PM.xcresult', scheme: 'app' }
```

Comment:
Maybe worth using a test path with even more special characters, including single quotes?

e.g.
```suggestion
          options = { project: "./scan/examples/standard/app.xcodeproj", result_bundle_path: './my_test_output/Alice\'s Man$ion 2024-09-30 at 5.10.35 PM.xcresult', scheme: 'app' }
```

Reply:
I have also updated the tests with a few other common special characters. Let me know if you have any additional suggestions.